cmd/storer2elastic/dataprocessor/tpsBenchmarkUpdater.go

```diff
@@ -2,23 +2,23 @@ package dataprocessor
 
 import (
 	"github.com/ElrondNetwork/elrond-go/core/check"
-	"github.com/ElrondNetwork/elrond-go/core/indexer"
 	"github.com/ElrondNetwork/elrond-go/core/statistics"
 	"github.com/ElrondNetwork/elrond-go/data/block"
+	"github.com/ElrondNetwork/elrond-go/process"
 	"github.com/ElrondNetwork/elrond-go/sharding"
 )
 
 const numMillisecondsInASecond = 1000
 
 type tpsBenchmarkUpdater struct {
 	tpsBenchmark   statistics.TPSBenchmark
-	elasticIndexer indexer.Indexer
+	elasticIndexer process.Indexer
```

Comment:
you could simply redefine the interface in this package as well and add only the needed functions.

Reply:
since the definition of the Indexer interface already depends on the process package (for example, method `SetTxLogsProcessor(txLogsProc process.TransactionLogProcessorDatabase)`) there is no real  benefit of redefining the Indexer interface here (or in consensus package). 
In this case I do not have a strong opinion against or in favor of redefining the interface.